File: back/DevC/src/main/java/com/back/devc/domain/member/mypage/dto/UpdateMyProfileRequest.java
package com.back.devc.domain.member.mypage.dto;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;

public record UpdateMyProfileRequest(

        @NotBlank(message = "닉네임은 필수입니다.")
        @Size(max = 50, message = "닉네임은 50자 이하여야 합니다.")
        String nickname
) {
}

File: back/DevC/src/main/kotlin/com/back/devc/domain/member/mypage/dto/UpdateMyProfileRequest.kt
package com.back.devc.domain.member.mypage.dto

import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.Size

data class UpdateMyProfileRequest(
    @field:NotBlank(message = "닉네임은 필수입니다.")
    @field:Size(max = 50, message = "닉네임은 50자 이하여야 합니다.")
    val nickname: String,
)